### PR TITLE
Installation of Brakeman

### LINKED ISSUE #47: As a developer, I want failed brakeman notices to fail my travis tests

### PR DESCRIPTION
**Pull request by dendenvitorio**
resolves #47

### CODE REVIEW

**Comment by pachonk** [comment]
Hi Denice, couple of things:

1. Can you make travis test brakeman too?
1. Make sure to include `resolves #47` in the main description of PR to link the issue and make it autoclose when this PR is pulled in.

**Comment by dendenvitorio** [comment]
Hi Alex,

1. Could you please elaborate what you mean by that?
2. Yeah, sure! thanks!

**Comment by pachonk** [comment]
https://github.com/emerald-rubies/tenacious/blob/master/.travis.yml is the configuration file
You should be able to add something like:
```
script:
  - bundle exec rake
  - bundle exec brakeman
```
the the bottom of the file to test this on travis.

**Comment by pachonk** [comment]
Travis is the tool that automatically tests our code using rspec: https://travis-ci.org/emerald-rubies/tenacious

File: .travis.yml [hunk 1]
```diff
@@ -4,4 +4,6 @@ rvm:
   - 2.4.1
 before_script:
   - bin/setup
-
+script:
+  - bundle exec rake
+  - bundle exec brakeman
```
Comment (pachonk) [inline .travis.yml]: new line at EOF.

Comment (pachonk) [inline .travis.yml]: Just did research, this needs to be `bundle exec brakeman -z` sorry!

Comment (dendenvitorio) [inline .travis.yml]: ok! hold on

File: .travis.yml [hunk 2]
```diff
@@ -5,3 +5,7 @@ rvm:
 before_script:
   - bin/setup
 
+script:
+  - bundle exec rake
+  - bundle exec brakeman -z
+
```
Comment (pachonk) [inline .travis.yml]: Now there's spaces here. Should be a 100% blank line :/ sorry!

**Pull request merged**